src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -213,11 +198,16 @@ function DepositAndEarningsCard({
           return acc.plus(
             new BigNumber(item.amount)
               .multipliedBy(tokenInfo?.priceUsd ?? 0)
-              .dividedBy(tokenInfo?.priceUsd ?? 1)
+              .dividedBy(depositTokenInfo?.priceUsd ?? 1)
           )
         }, new BigNumber(0))
     )
-  }, [balance, earningItems, earningItemsTokenInfo])
+  }, [balance, earningItems, earningItemsTokenInfo, depositTokenInfo])
+
+  const totalDepositBalanceInLocalCurrency =
+    useDollarsToLocalAmount(
+      totalDepositBalanceInCrypto.multipliedBy(depositTokenInfo?.priceUsd ?? 0)
```

Comment:
I like the use of `totalDepositBalanceInCrypto * priceUsd` here instead of another filter / reduce.